internal/service.go

```diff
@@ -355,14 +206,7 @@ func (h *HauserService) WriteBundleToCSV(bundleID int, tableColumns []string, cs
 }
 
 // WriteBundleToJson writes the bundle corresponding to the given bundleID to a Json file
-func (h *HauserService) WriteBundleToJson(bundleID int, filename string) (bytesWritten int64, err error) {
-	stream, err := h.getExportData(bundleID)
-	if err != nil {
-		log.Printf("Failed to fetch bundle %d: %s", bundleID, err)
-		return 0, err
-	}
-	defer stream.Close()
-
+func (h *HauserService) WriteBundleToJson(stream io.Reader, filename string) (bytesWritten int64, err error) {
```

Comment:
Correct. The support for `StorageOnly is tenuous. You don't get consistency across exports because we don't store the fields anywhere. I could add support for that, but that feel out of scope for this PR since it's an existing problem.

And I should definitely remove this function. I just missed it.